data/migrations/v1.5.0-planet_osm_point.sql

```diff
@@ -1,11 +1,13 @@
 -- Add new shop types
 UPDATE planet_osm_point
   SET mz_poi_min_zoom = mz_calculate_min_zoom_pois(planet_osm_point.*)
-  WHERE mz_poi_min_zoom <> mz_calculate_min_zoom_pois(planet_osm_point.*)
+  WHERE (mz_poi_min_zoom <> mz_calculate_min_zoom_pois(planet_osm_line.*)
+    AND mz_poi_min_zoom <> mz_calculate_min_zoom_pois(planet_osm_polygon.*))
```

Comment:
This looks like you're comparing `mz_poi_min_zoom` on points to lines and polygons, which I think is not valid SQL. What did you want to happen here?